wordpress-mobile/WordPress-iOS
# [Insights Management] Add `Customize your insights` card

Fixes #12244 

To test:

---
TIP: If you'd like to force the Customize card to show without reinstalling the app: 
- In `SiteStatsInsightsTableViewController:loadInsightsFromUserDefaults`, set `hideCustomizeCard` to `false`.
- When you navigate away from Stats and back, the card will show.

NOTE: The image is currently using 1/2/3x PNGs. I've asked @SylvesterWilmott for a PDF version, and will update it when received.

---
- Go to Insights.
- Verify the customize card is displayed at the top.

<img width="400" alt="customize" src="https://user-images.githubusercontent.com/1816888/62329015-e94db980-b471-11e9-8d47-2b15fa60d695.png">

---
- Select 'Try it now'.
- Verify an alert is displayed (it's temporary, since the IM view doesn't exist yet).

<img width="300" alt="alert" src="https://user-images.githubusercontent.com/1816888/62329099-2a45ce00-b472-11e9-986d-85c51e9c20a6.png">

---
- Select 'Dismiss'.
- Verify the Customize card is dismissed.
  - NOTE: I banged my head for a while trying to animate that. I'll come back to it later.
- Verify the Customize card is not displayed again.
  - Navigate away from Stats and return.
  - Select another site.
  - Relaunch the app.

---
- Reinstall the app.
- Verify the Customize card is displayed.

---
- Disable the `statsInsightsManagement` feature flag.
- Verify the Customize card is not displayed.

---
Update release notes:
